src/main/java/world/bentobox/bentobox/listeners/flags/protection/BlockInteractionListener.java

```diff
@@ -213,6 +213,7 @@ private void checkClickedBlock(Event e, Player player, Location loc, Material ty
         case SPRUCE_FENCE_GATE:
             checkIsland(e, player, loc, Flags.GATE);
             break;
+        case FURNACE_MINECART:
```

Comment:
Have you tested it? I doubt "FURNACE_MINECART" is actually a block...

Reply:
So, there were also CHEST_MINECART used above, so I added FURNACE_MINECART here.

Reply:
You're right!